src/postgres/sandbox.yaml

```diff
@@ -0,0 +1,6 @@
+name: test-postgres
```

Comment:
Did you mean to check this in?

Reply:
yes, I think it will help facilitate development of variants to have artefacts in the sources that are used in testing.